### DISCUSSION TITLE
Tooltips for html and markdown elements

### DISCUSSION THREAD

**Discussion by 1081** [zauberzeug/nicegui#2193] [Ideas / Feature Requests]
Please add the possibility to add a tooltip for html and markdown elements.

Example: Custom HTML-label with subscript for a number input in a scientific application.
![image](https://github.com/zauberzeug/nicegui/assets/11081331/fcda8e48-7084-48fc-abd5-8dc9f5cb8547)

See also discord [question](https://discord.com/channels/1089836369431498784/1185512982466277457/1185512982466277457):

**Comment by falkoschindler** [comment]
Interesting, I never thought about tooltips in `ui.html`! Currently it doesn't work, because `ui.html` (as well as `ui.markdown` and `ui.mermaid`) are `ContentElement`s that can't nest other NiceGUI elements, but set their inner HTML via the `content` property. So we basically need to add support for child elements in a content element. Not sure if and how this is possible. But I'll think about it. If anyone has an idea, I'm all ears. 🙂

**Comment by falkoschindler** [comment]
It seems pretty complicated to add support for nested elements in `ContentElement`s like `ui.html` and `ui.markdown`. But there's a pretty simple workaround:
```py
with ui.element():
    ui.html('This is <u>HTML</u>...')
    ui.tooltip('...with a tooltip!')
```

Or:
```py
with ui.element().tooltip('...with a tooltip!'):
    ui.html('This is <u>HTML</u>...')
```

This basically wraps the HTML element in a container with a tooltip.
